pkg/storage/filesystem/btrfs_ci_test.go

```diff
@@ -38,11 +38,17 @@ func (d TestDevices) Loops() Devices {
 	mgr := lsblkDeviceManager{
 		executer: executerFunc(run),
 	}
-	for _, loop := range d {
+	for loopTempPath, loop := range d {
+		size, err := FilesUsage(loopTempPath)
+		if err != nil {
+			panic(err)
+		}
```

Comment:
require error not nil please, don't panic

Reply:
also require no error replaces the if clause you shouldn't add it inside the if body